### ISSUE TITLE
Ember.A issues

### ISSUE THREAD

**Issue by rwwagner90** [DockYard/eslint-plugin-ember-suave#14]
Whenever I use `A()` I get errors like this:

```
A function with a name starting with an uppercase letter should only be used as a constructor
```

**Comment by brzpegasus** [comment]
Does it say which rule is triggering that?

**Comment by alexlafroscia** [comment]
It probably comes from the `new-cap` rules.  I added the following to my ESLint config at work to get around it:

```js
{
  rules: {
    ...
    'new-cap': ['error', {
      // Lower-case variables that can be used with `new`
      'newIsCapExceptions': [
        'hash' // RSVP.hash
      ],
      // Capital variables that can be used without `new`
      'capIsNewExceptions': [
        'A', // Ember.A
        'Deferred' // $.Deferred
      ]
    }],
  }
}
```

**Comment by rwwagner90** [comment]
Yeah, it's from `new-cap`. Can we add that modified `new-cap` rule to our config @alexlafroscia?

**Comment by alexlafroscia** [comment]
Yeah, we probably could just add that to the recommended config.  Thoughts @brzpegasus?

**Comment by brzpegasus** [comment]
> ```js
'newIsCapExceptions': [
  'hash' // RSVP.hash
]
```

When/why would you call `RSVP.hash` with `new`?

**Comment by alexlafroscia** [comment]
I dunno, but I found it in our codebase ¯\\_(ツ)_/¯

**Comment by rwwagner90** [comment]
@brzpegasus I'm not calling any of them explicitly with the `new` keyword. It just flags issues like:

`myArray: A([])`

**Comment by brzpegasus** [comment]
Yeah, I'll add `A` because it's not a constructor, just wasn't sure why `RSVP.hash` was included in the above config as something that could be called with `new`.

**Comment by rwwagner90** [comment]
@brzpegasus want me to do a PR to include the config @alexlafroscia mentioned above?
